src/main/java/me/kavin/piped/utils/ChannelHelpers.java

```diff
@@ -95,6 +99,23 @@ public static SyndEntry createEntry(Video video, Channel channel) {
 
         entry.setContents(List.of(thumbnail, content));
 
+        // the Media RSS content for embedding videos starts here
+        // see https://www.rssboard.org/media-rss#media-content
+
+        String playerUrl = Constants.FRONTEND_URL + "/embed/watch?v=" + video.getId();
```

Comment:
I don't think this URL is right, it should just be `/embed/`

Reply:
Good catch!